conjur/api/client.py

```diff
@@ -49,15 +48,15 @@ class Client():
     # what parameters are allowed
     # pylint: disable=too-many-arguments,too-many-locals,too-many-branches,line-too-long,try-except-raise,too-many-statements
     def __init__(self,
-                 account=None,
+                 account:str=None,
                  api_key=None,
                  ca_bundle=None,
```

Comment:
what about api_key and ca_bundle?  they are both strings

Reply:
done